Makefile

```diff
@@ -0,0 +1,15 @@
+NODE_BIN=./node_modules/.bin
+
+requirements:
+	npm install
+
+develop: requirements
+
+test: requirements
+	$(NODE_BIN)/karma start --single-run
+
+test-dev: requirements
+	$(NODE_BIN)/karma start
+
+quality: requirements
+	$(NODE_BIN)/jshint lib/ test/
```

Comment:
For this, I'll copy from the toolkit. I can update later if we get a standard somewhere. Thanks for the help @dsjen @andy-armstrong